app/controllers/annotations_controller.rb

```diff
@@ -119,4 +49,28 @@ def init_annotation(text, question, org, type)
     return annotation
   end
 
+  def process_changes(hash, input, type)
```

Comment:
I don't see the need of this private function unless it is re-used in several actions.

Reply:
its called twice within the `admin_update` action

Reply:
I did not notice it was called twice within admin_update, in that case, it makes sense to have a private method. I don't see the case to using Proc or its specialised Lambda to this purpose since the method process_changes is not using any variable defined at the scope of admin_update